tests/ui/items/test_items_api.py

```diff
@@ -231,5 +232,23 @@ def test_items_availability(item_type_missing_martigny,
     assert item.available
     assert len(item.availability_text) == 1  # only default value
 
+    # test availability and availability_text for an issue
+    item['type'] = TypeOfItem.ISSUE
+    item['enumerationAndChronology'] = 'duummy'
```

Comment:
Is it actually `duumy` with a double `u`?

Reply:
keyboard mistake ! Or maybe ICC problem ? (https://fr.wikipedia.org/wiki/Jargon_informatique#I)